components/TimeSeriesList.client.vue

```diff
@@ -3,7 +3,6 @@
     <TimeSeries
       v-for="(_, seriesId, index) in appStore.timeSeriesData"
       :key="seriesId"
-      :all-charts="charts"
```

Comment:
Why was this not needed any more?

Reply:
I forgot to remove it in a refactor a few weeks ago. It doesn't do anything.